gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/panels/AssignmentColumnHeaderPanel.java

```diff
@@ -153,17 +160,7 @@ public void onClick() {
 		add(new AttributeModifier("data-category-order", assignment.getCategoryOrder()));
 
 		// menu
-		final WebMarkupContainer menu = new WebMarkupContainer("menu") {
-			private static final long serialVersionUID = 1L;
-
-			@Override
-			public boolean isVisible() {
-				if (role != GbRole.INSTRUCTOR) {
```

Comment:
Very minor but this check could be similar to the ones below?
return role == GbRole.TA

Will merge so this doesn't get held up, may be it ca be modified after.